Update pyproject.toml

debpal opened the pull request:
This pull request includes modifications to license formats, addition of PyPI classifiers, and new keywords. Please review it.

The new license format is required as per [PyPA guidelines](https://packaging.python.org/en/latest/guides/writing-pyproject-toml/); otherwise, a warning is raised when running `python -m build`. Additionally, we should specify the minimum supported Python version as 3.9.

Even though the previous pull request has been merged, my fork appears as "3 commits ahead" of the upstream. This is likely due to GitHub’s merge behavior (e.g., squash or rebase). If needed, I can reset my fork to match upstream to avoid confusion.

dhadka commented on pyproject.toml:
```diff
@@ -9,15 +9,34 @@ authors = [
+    "Development Status :: 5 - Production/Stable",
+    "Programming Language :: Python :: 3",
+    "Programming Language :: Python :: 3.13",
+	"Programming Language :: Python :: 3.12",
```
```suggestion
    "Programming Language :: Python :: 3.12",
```

dhadka reviewed (commented):
Left some minor suggestions.  It looks like the new `license` format isn't compatible with 3.8 causing the CI to fail, so we will need to bump the minimum version here and in the workflow files.

debpal commented:
@dhadka I didn’t intend to change your original setup, but thank you for the suggestions. The following updates have been made:

- Renamed `COPYING` to `LICENSE` to avoid confusion.
- Updated license identifier to `GPL-3.0-or-later` instead of `GPL-3.0-only`.
- Removed `Python 3.8` from the GitHub workflow and set `Python 3.9` as the minimum version.
- All tests are now passing.
- Added badges to the `README.md` for improved visibility. Some badge statuses will update automatically after your next release to the `PyPI` repository.
